Assets/Scripts/NPC/Idle.cs

```diff
@@ -0,0 +1,31 @@
+using UnityEngine;
+
+public class Idle : SmallEnemiesBaseState
+{
+    private float _currentWaitTime;
+    [SerializeField, Range(0, 30)] private float waitTime;
+
```

Comment:
```suggestion
{
    [SerializeField, Range(0, 30)] private float waitTime;
    
    private float _currentWaitTime;
    
```
Persoonlijk heb ik liever SerializeField boven true private.